packages/debugger/.env.sample

```diff
@@ -0,0 +1,7 @@
+# Example: FARCASTER_DEVELOPER_MNEMONIC="candy gibraltar foxtrot kilo barnacles foxes ..."
+# (OPTIONAL) Needed for the debugger to create a real Farcaster signer. Get this by exporting your seed phrase from the Warpcast app. Don't share that seed phrase with anyone.
+FARCASTER_DEVELOPER_MNEMONIC=
+# (OPTIONAL) Only needed for the debugger to create a real Farcaster signer. Get this by visiting your Warpccast profile, pressing the kebab (three dots) menu and then "About" and then your fid should be there.
+# Example: FARCASTER_DEVELOPER_FID=1214
+FARCASTER_DEVELOPER_FID=
```

Comment:
@davidfurlong @stephancill how these env variables affect the debugger? Are they necessary?